### DISCUSSION TITLE
Setting for player death listener priority

### DISCUSSION THREAD

**Discussion by AxDevv** [Zrips/CMI#7180] [Enhancement]
Hello I have a problem with other plugin that manages the reappearance point, CMI takes over after a server restart, that's why I propose this feature, to set up a system of priority spawn for player (death listener priority)

**Comment by Zrips** [comment]
About which feature are you talking here exactly? Respawn->PriorityOrder? And generally speaking changing priority might not really change outcome as it would still be processed in same way, so I need more information here to get better idea.

**Reply by AxDevv** [reply]
I use the SavageGraveyard plugin, it uses a priority system, but even in Highest, when I restart the server, the respawn point of cmi takes over, I have to reload the graveyard plugin for the priority to go up ^^
